### PR TITLE
TELCODOCS-1415: Add information about status.state.AtLatestKnown for Subscription CRs

### PR DESCRIPTION
**Pull request by amolnar-gh**
Version(s): 4.12, 4.13, 4.14

Issue: https://issues.redhat.com/browse/TELCODOCS-1415

Link to docs preview: https://62928--docspreview.netlify.app/openshift-enterprise/latest/scalability_and_performance/ztp_far_edge/cnf-talm-for-cluster-upgrades#talo-about-subscription-crs_cnf-topology-aware-lifecycle-manager

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by jc-rh** [comment]
I am adding an operator policy example to the upstream repo. Do we need to include a reference to it in this doc section?

**Comment by jc-rh** [comment]
https://github.com/openshift-kni/cluster-group-upgrades-operator/blob/main/samples/operator-policy.yaml

File: modules/cnf-topology-aware-lifecycle-manager-about-subscription-crs.adoc [hunk 1]
```diff
@@ -0,0 +1,71 @@
+// Module included in the following assemblies:
+// Epic CNF-2600 (CNF-2133) (4.10), Story TELCODOCS-285
+// * scalability_and_performance/cnf-talm-for-cluster-upgrades.adoc
+
+:_content-type: PROCEDURE
+[id="talo-about-subscription-crs_{context}"]
+= Configuring Operator subscriptions for managed clusters that you install with {cgu-operator}
+
+{cgu-operator} can only approve the install plan for an Operator, if the `Subscription` CR of the Operator contains a valid `status` field. You can use the following fields:
```
[the remark this answers is not in the document]
Comment (skrthomas) [reply modules/cnf-topology-aware-lifecycle-manager-about-subscription-crs.adoc:9]: I don't think the comma is needed here

**Review by skrthomas — COMMENTED** [review]
A couple of comments/suggestions for ya :) Nice work!

**Pull request merged**